### PR TITLE
DEV: enforces link-rel-noopener linting rule

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

**Comment by jjaffeux** [comment]
@eviltrout this one should be quite safe

File: app/assets/javascripts/discourse/templates/user.hbs [hunk 1]
```diff
@@ -104,13 +104,16 @@
               </h3>
             {{/if}}
 
+
+            AAAAAA
```
Comment (CvX) [inline app/assets/javascripts/discourse/templates/user.hbs]: 👀

File: app/assets/javascripts/discourse/templates/user/summary.hbs [hunk 2]
```diff
@@ -77,10 +77,11 @@
         <ul>
           {{#each model.links as |link|}}
             <li>
+              {{!-- template-lint-disable  --}}
               <a class='domain'
                  href='{{unbound link.url}}'
                  title='{{unbound link.title}}'
-                 rel='{{unless user.removeNoFollow 'nofollow ugc noopener'}}'
+                 rel='{{unless user.removeNoFollow 'nofollow ugc noopener' 'noopener'}}'
```
Comment (CvX) [inline app/assets/javascripts/discourse/templates/user/summary.hbs]: This should work, right?

```suggestion
                 rel='noopener {{unless user.removeNoFollow 'nofollow ugc'}}'
```

Comment (CvX) [reply app/assets/javascripts/discourse/templates/user/summary.hbs]: (Same with the other `template-lint-disable` instances)

File: app/assets/javascripts/discourse/templates/user.hbs (hunk 1 above)
Comment (ZogStriP) [reply app/assets/javascripts/discourse/templates/user.hbs]: BBBBBB

Comment (jjaffeux) [reply app/assets/javascripts/discourse/templates/user.hbs]: damn was sure I had removed it

File: app/assets/javascripts/discourse/templates/user/summary.hbs (hunk 2 above)
Comment (jjaffeux) [reply app/assets/javascripts/discourse/templates/user/summary.hbs]: yes I think it would, will try :+1:

Comment (jjaffeux) [reply app/assets/javascripts/discourse/templates/user/summary.hbs]: Sadly it still errors in the linting, but I changed it to your syntax as it still prevents duplicating noopener

**Comment by eviltrout** [comment]
Merging, I think the plugin failure is not related.

**Pull request merged**